package.json

```diff
@@ -16,11 +16,12 @@
   "author": "accordproject.org",
   "license": "Apache-2.0",
   "dependencies": {
-    "@accordproject/cicero-core": "0.10.2",
-    "@accordproject/cicero-template-library": "git+https://git@github.com/accordproject/cicero-template-library.git",
-    "@accordproject/ergo-compiler": "0.6.3",
+    "@accordproject/cicero-core": "0.12.0",
+    "@accordproject/cicero-template-library": "git+https://git@github.com/accordproject/cicero-template-library.git#cicero-0.12",
```

Comment:
What is this dependency for? Is it really a dev dependency?

Reply:
Missed this comment :( Let me review.

Reply:
Cleaned up in https://github.com/accordproject/template-studio/pull/102